### PR TITLE
create page to display employees in the portal, create links to other…

### PR DESCRIPTION
**Pull request by carloso0114**
# Description
- Portal page
- Create get routes to display employees information, and links to edit current employee, links to create new employee, and new stores. 

## Testing

Navigate to /portal 
and you would be able to see the information.

## Screenshots

image

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -77,10 +77,14 @@ body {
...
 
-.btn-success {
-  background-color: #11763B;
+.btn-green {
+  background-color: $green-100;
+}
+
+.btn-green:hover {
+  background-color: $green-400;
 }
```
Comment (xManuel007) [inline app/assets/stylesheets/application.scss:90]: Missing blank line here at final

**Review by Reiyirak — APPROVED** [review]
LGTM

File: app/views/employees/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Editando </h1>
+<h1><%= @employee.name %></h1>
```
Comment (xManuel007) [inline app/views/employees/edit.html.erb:2]: Also here

File: config/routes.rb [hunk 3]
```diff
@@ -6,6 +6,7 @@
   resources :attendances
 
   resources :admins, only: [:new, :create]
-  resources :employees, only: [:index]
+  resources :employees, only: [:index, :edit, :new]
+  resources :stores, only: [:new]
   get 'portal', to: 'employees#index'
 end
```
Comment (xManuel007) [inline config/routes.rb:12]: And here

**Review by xManuel007 — CHANGES_REQUESTED** [review]

**Review by juperameza — CHANGES_REQUESTED** [review]
Just attend @xManuel007 comments

**Review by xManuel007 — APPROVED** [review]

**Review by juperameza — APPROVED** [review]
LGTM 
The "en construcción" files look weird to me because if you're not going to do it why did you create them but ig it's okay

**Pull request merged**